manager/client/main.go

```diff
@@ -0,0 +1,93 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	rpc_license "github.com/enfabrica/enkit/manager/rpc"
+	"google.golang.org/grpc"
+	"io"
+	"log"
+	"os"
+	"os/exec"
+	"os/user"
+	"strings"
+	"time"
+)
+
+func run(timeout time.Duration, cmd string, args ...string) {
+	ctx, cancel := context.WithTimeout(context.Background(), timeout)
+    defer cancel()
+    job := exec.CommandContext(ctx, cmd, args...)
+    err := job.Run()
+    if ctx.Err() == context.DeadlineExceeded {
+		log.Fatalf("Job failed to complete after %d seconds: %s %s \n", timeout, cmd, strings.Join(args, " "))
+    }
+    if err != nil {
+        log.Fatalf("Job \"%s %s\" failed with error %s", cmd, strings.Join(args, " "), err)
+    } else {
+		log.Printf("Job completed successfully: %s %s \n", cmd, strings.Join(args, " "))
+    }
+}
+
+func polling(client rpc_license.LicenseClient, username string, quantity int32, vendor string, feature string,
+	cmd string, args ...string) {
+	timeout := 1800 * time.Second
```

Comment:
It would be a really good idea to use a context.WithTimeout here instead of setting it as a manual timeout. 
Reason: -> you can pass the context down a chain of async commands and they're all reading from the same timeout.
                  also, it looks clearer and can be used with grpc-web